screens/Scan/ScanScreen.tsx

```diff
@@ -16,10 +16,13 @@ import {BannerNotificationContainer} from '../../components/BannerNotificationCo
 import {SharingStatusModal} from './SharingStatusModal';
 import {SvgImage} from '../../components/ui/svg';
 import {LocationPermissionRational} from './LocationPermissionRational';
+import { FaceVerificationAlertOverlay } from './FaceVerificationAlertOverlay';
+import { useSendVcScreen } from './SendVcScreenController';
 
 export const ScanScreen: React.FC = () => {
   const {t} = useTranslation('ScanScreen');
   const controller = useScanScreen();
+  const controller_scan = useSendVcScreen();
```

Comment:
can we rename both the controller names with some other names,
may be  scanScreenController, sendVcScreenController

Reply:
Done